airbyte-cdk/python/unit_tests/sources/test_source.py

```diff
@@ -425,8 +425,8 @@ def test_source_config_no_transform(abstract_source, catalog):
     records = [r for r in abstract_source.read(logger=logger_mock, config={}, catalog=catalog, state={})]
     assert len(records) == 2 * 5
     assert [r.record.data for r in records] == [{"value": 23}] * 2 * 5
-    assert http_stream.get_json_schema.call_count == 1
-    assert non_http_stream.get_json_schema.call_count == 1
+    assert http_stream.get_json_schema.call_count == 5
```

Comment:
this assert verified that we used the cached transformer and schema instead of calling `get_json_schema`.
We now call `get_json_schema` more often, but the value is cached so the behavior is effectively the same

Reply:
done https://github.com/airbytehq/airbyte/pull/18572/commits/0cbd28e88f6a3c41b6da9212ba37e22bb9456bfe